optimum/neuron/__init__.py

```diff
@@ -12,12 +12,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
+from typing import TYPE_CHECKING
 
-from .hf_argparser import TrainiumHfArgumentParser
-from .trainers import Seq2SeqTrainiumTrainer, TrainiumTrainer
-from .utils.training_utils import patch_transformers_for_neuron_sdk
+from transformers.utils import _LazyModule
 
 
-if not os.environ.get("DISABLE_TRANSFORMERS_PATCHING", False):
-    patch_transformers_for_neuron_sdk()
+_import_structure = {
+    "hf_argparser": ["TrainiumHfArgumentParser"],
+    "trainers": ["TrainiumTrainer", "Seq2SeqTrainiumTrainer"],
+    "utils": ["is_neuron_available", "is_neuronx_available", "patch_transformers_for_neuron_sdk"],
```

Comment:
Here we call `patch_transformers_for_neuron_sdk`, we dont import it.
But I think it does not do anything anyways.

Reply:
Oh right. Put it outside the import code.